packages/core/src/lib/light_push/index.ts

```diff
@@ -172,6 +170,8 @@ class LightPush extends BaseProtocol implements ILightPush {
     });
 
     const results = await Promise.allSettled(promises);
+
+    // TODO: handle renewing faulty peers with new peers
```

Comment:
is there an issue open for this?

Reply:
the parent issue #1463 accounts for this

Reply:
Can you, please, mention this issue in the TODO so we don't need to guess later on?

Reply:
> Can you, please, mention this issue in the TODO so we don't need to guess later on?

done